src/app/dashboard/users/page.tsx

```diff
@@ -133,14 +171,34 @@ async function getUsers(
     }),
   )
 
-  users.map((user, i) => {
+  listedUsers.map((user, i) => {
     user.requests = usersRequestsCounts[i]?.requests || 0
     user.movies_plays_count = usersPlaysAndDurations[i].movies_plays_count
     user.shows_plays_count = usersPlaysAndDurations[i].shows_plays_count
     user.audio_plays_count = usersPlaysAndDurations[i].audio_plays_count
   })
 
-  return users
+  return listedUsers
+}
+
+async function getStatsWithLoggedInUser(
+  userId: number,
+  users: TautulliUserItemRow[],
+  numberOfUsers: number,
+) {
+  let slicedUsers = users.slice(0, numberOfUsers)
+  const loggedInUserRank = users.findIndex((user) => user.user_id == userId)
+  const loggedInUser =
```

Comment:
What's the use-case where `getInactiveUserInTimePeriod` would actually run? Considering we already have all the users in the server available in the passed `users` object.

Reply:
`users` does not contain accounts who have been in active in the given time period. For example, _User A_ has not watched anything in the last week -> `users` does not contain data of _User A_